client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -92,6 +110,12 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 						/>
 					</CityZipRow>
 
+					<TextControl
+						label={ __( 'Email address', 'woocommerce-admin' ) }
+						value={ getProfileEmail() }
+						onChange={ ( value ) => updateProfileEmail( value ) }
```

Comment:
Could this be simplified?
```suggestion
						onChange={ updateProfileEmail }
```